### ISSUE TITLE
New SD card,format and FBI (second time?)

### ISSUE THREAD

**Issue by PrometheusG** [Plailect/Guide#326]
hi!
I will start doing part 4 of plailect's guide today after after almost 1 month of completing part 3.
I am using a 2ds and I bought a new sd card 32GB at part 3 to continue.

It seems that I have formated the new sd card following this guide: https://github.com/Plailect/Guide/issues/231 at post #6 by @nickthegamer5 (which he helped me)
Will that be a problem?
Because I saw on gbatemp.net that I have to finish the guide first,then change to a new sd card

Also If I format rednand 3 times just to be sure(because I did format the new sd card with emunand 9 and I don't know if it counts) will the rednand and sysnand be linked?

Also the guide at part 4 in instructions step II-Injecting FBI, says to inject FBI to H&S but I have already done that at 9.2.0 Update Page of Plailect's Guide.Must I do it again?

thank you!

**Comment by Plailect** [comment]
Formatting it extra times won't affect anything.

If you can open FBI, you don't have to inject it again.

**Comment by PrometheusG** [comment]
Thanks! I just finished your guide!you are awesome!

Although I have a little problem now:I injected back the h&s app(following part 5 section VI,last 4 steps) that I had from the 9.2.0 update(I was on 7.2.0.12E and used rxtools 3.0 beta on june 28) and it is working but the icon is still FBI.
FBI.cia is installed too so I have 2 fbi's but the h&s app shows on the upper screen that it is h&s and the cia file shows that it is indeed FBI

what should I do?
